modules/external-api/src/main/java/org/opencastproject/external/endpoint/SeriesEndpoint.java

```diff
@@ -271,10 +270,10 @@ public Response getSeriesList(@HeaderParam("Accept") String acceptHeader, @Query
             case SeriesIndexSchema.CONTRIBUTORS:
               query.sortByContributors(criterion.getOrder());
               break;
-            case SeriesIndexSchema.CREATOR:
+            case SeriesIndexSchema.ORGANIZERS:
               query.sortByOrganizers(criterion.getOrder());
               break;
-            case EventIndexSchema.CREATED:
+            case SeriesIndexSchema.CREATED_DATE_TIME:
```

Comment:
I'm gonna rain on your parade here and tell you that this probably needs a new API version, because this is not backwards compatible. :( On the other hand, you could also ignore the API for now...